examples/deeplab/yamls/mcloud_run.yaml

```diff
@@ -7,8 +7,9 @@ integrations:
 - integration_type: git_repo
   git_repo: mosaicml/examples
 command: |
+  cd examples
+  pip install -e ".[deeplab]"
   cd examples/deeplab
-  pip install -r requirements.txt
```

Comment:
you could instead follow what llm / bert do and add it to the integrations. [eg bert does](https://github.com/mosaicml/examples/blob/main/examples/bert/yamls/main/mcloud_run_a100_40gb.yaml#L21):
```
integrations:
- integration_type: git_repo
  git_repo: mosaicml/examples
  # git_branch: # use your branch
  # git_commit: # OR use your commit hash
  pip_install: -e .[bert]
command: |
  cd examples/examples/bert
  composer main.py /mnt/config/parameters.yaml
```